linodecli/completion.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/local/bin/python3
+"""
+Contains any code relevant to generating/updating shell completions for linode-cli
+"""
+
+from string import Template
+
+def bake_completions(ops):
+    """
+    Given a baked CLI, generates and saves a bash completion file
+    """
+    print("Baking bash completions...")
+    if "_base_url" in ops:
+        del ops["_base_url"]
+    if "_spec_version" in ops:
+        del ops["_spec_version"]
+    rendered = get_bash_completions(ops)
+    with open("linode-cli.sh", "w", encoding="utf-8") as bash_f:
+        print("Writing file...")
+        bash_f.write(rendered)
+
+def get_completions(ops, help_flag, action):
+    """
+    Handle shell completions based on `linode-cli completion ____`
+    """
+    if help_flag or not action:
+        return """linode-cli completion [SHELL]
```

Comment:
Use triple quotes here will make the `str` inconsistent with the one before refactoring. Maybe consider use multiple `str` with newline character?

```python
return (
    "str1\n",
    "str2"
)
```